contracts/Core/CollectionManager.sol

```diff
@@ -238,8 +240,8 @@ contract CollectionManager is Initializable, CollectionStorage, StateManager, Co
         uint16[] memory jobIDs
     ) external onlyRole(COLLECTION_MODIFIER_ROLE) notState(State.Commit, buffer) {
         require(collectionID <= numCollections, "Collection ID not present");
-        require(collections[collectionID].active, "Collection is inactive");
         require(tolerance <= maxTolerance, "Invalid tolerance value");
+        require(jobIDs.length > 0, "job not present");
```

Comment:
@yohanelly95 Can we keep the same error message as we have in createCollection for `jobIDs.length` check?